# HG changeset patch
# User jwe
# Date 745019316 0
#      Tue Aug 10 21:48:36 1993 +0000
# Node ID 19b973bc5464bbb7aa6754f1b8c86b7c273ccc74
# Parent  9239bd2337bb20bd65733f751c214a67fbe022a6
[project @ 1993-08-10 21:48:36 by jwe]
(general_functions): Add givens to the list.

diff --git a/src/builtins.cc b/src/builtins.cc
--- a/src/builtins.cc
+++ b/src/builtins.cc
@@ -339,18 +339,25 @@ where y and x are vectors.\n\n", },
     "solve NLPs\n", },
 
   { "ftell", 2, 1, builtin_ftell,
     "position = ftell ('filename' or filenum): returns the current file position\n", },
 
   { "getenv", 2, 1, builtin_getenv,
     "get environment variable values\n", },
 
+  { "givens", 3, 2, builtin_givens,
+    "Givens rotation:\n\
+  G = givens(x,y) returns an orthogonal matrix G = [c s; -conj(s) c] such\n\
+    that G[x;y] = [*;0]  (x,y scalars)\n\
+  [c,s] = givens(x,y) returns the (c,s) values themselves.",},
+  
+
   { "hess", 2, 2, builtin_hess,
-    "Hessenburg decomposition\n",},
+    "Hessenberg decomposition\n",},
 
   { "home", 1, 0, builtin_clc,
     "clear screen\n", },
 
   { "input", 3, 1, builtin_input,
     "input('prompt' [,'s']): prompt user for [string] input\n", },
 
   { "ifft", 2, 1, builtin_ifft,
